.github/workflows/build/Dockerfile.ubuntu-2204

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:22.04
+
+ARG uid=1000
+ARG user=indy
+
+RUN apt-get update -y && apt-get install -y \
+    # common stuff
+    git \
+    apt-transport-https \
+    apt-utils\
+    wget\
+    curl\
+    jq\
+    gnupg
+
+# ========================================================================================================
+# Update repository signing keys
+# --------------------------------------------------------------------------------------------------------
+    # Hyperledger
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 9692C00E657DDE61 && \
+    # Sovrin
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
+# ========================================================================================================
+
+# Plenum
+#  - https://github.com/hyperledger/indy-plenum/issues/1546
+#  - Needed to pick up rocksdb=5.8.8
+#RUN echo "deb https://hyperledger.jfrog.io/artifactory/indy focal dev"  >> /etc/apt/sources.list && \
+#    echo "deb https://repo.sovrin.org/deb bionic master" >> /etc/apt/sources.list && \
+#    echo "deb https://repo.sovrin.org/sdk/deb bionic master" >> /etc/apt/sources.list
+
+# TODO change this to official repo
+RUN echo "deb [trusted=yes] http://209.141.41.82:8000/ packagedir/" >> /etc/apt/sources.list
```

Comment:
@reflectivedevelopment, Which repo is this?

Reply:
This is a temporary repo, until we get the jfrog packages installed.